contracts/examples/ping-pong-egld/tests/mandos_test.rs

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
Missing test:
Calling pong_all after at least 1 user already pong-ed.

Reply:
Added it.